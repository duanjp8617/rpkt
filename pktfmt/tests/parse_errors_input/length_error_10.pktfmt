packet FakeProto {
    header = [
        field_1 = Field {
            bit = 16, 
            repr=&[u8],
            gen=false
        },
        field_2 = Field {
            bit = 8, 
            gen=false
        },
        field_3 = Field {
            bit = 8, 
        }
    ],
    length = [
        header_len =field_1/4-2,
        payload_len =field_2*3,
    ]
}
