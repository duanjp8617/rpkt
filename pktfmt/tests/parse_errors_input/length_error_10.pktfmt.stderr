ast error
at /home/djp/pkt-rs/pktfmt/tests/parse_errors_input/length_error_10.pktfmt 17:21-31
17 |         header_len =field_1/4-2,
                         ^^^^^^^^^^^
field error 10:
the form of the algorithmic expression is too complex, only simple ones are supported
